databases/week2/QestionNo.3.txt

```diff
@@ -0,0 +1,22 @@
+/* Get all the tasks assigned to users whose email ends in @spotify.com*/
+
+	select * from  task
+	inner join user on task.status_id=user.id
+	where user.email like '%@spotify.com';
```

Comment:
Nice. One thing to think about is that this query will actually return duplicates if two users have `@spotify.com` in their email and also are collaborating on the same task(s). Can you think of a way to avoid the duplicate tasks?